src/errors/internal-modules-prefix.ts

```diff
@@ -0,0 +1,5 @@
+import semver from 'semver';
+
+const INTERNAL_MODULES_PREFIX = semver.gte(process.version, '15.0.0') ? 'node:' : 'internal/';
```

Comment:
`semver` is still in `devDependencies`. We can move it to `dependencies`, but I feel we do not really need `semver` here. A RegExp or an array of prefixes can do the job well.